sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/constants.py

```diff
@@ -14,6 +14,11 @@
 #: The port number is reserved for future transport mappings to these protocols.
 PORT = 5672
 
+# default port for AMQP over Websocket
+WEBSOCKET_PORT = 443
+
+# subprotocol for AMQP over Websocket
+AMQP_WS_SUBPROTOCOL = 'AMQPWSB10'
```

Comment:
Is this universal to AMQP? Or is this somehow Azure specific?

Reply:
it's universal 
http://docs.oasis-open.org/amqp-bindmap/amqp-wsb/v1.0/amqp-wsb-v1.0.html